mqtt/mqtt-broker/src/persist.rs

```diff
@@ -136,6 +145,211 @@ impl FileFormat for BincodeFormat {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct ConsolidatedStateFormat;
```

Comment:
Why keep both formats?

Reply:
I think we should keep it as a backup. The compiler will remove it anyways on release build.

Reply:
Lets not leave unused code. I may lead to confusion which variant we really use and why.